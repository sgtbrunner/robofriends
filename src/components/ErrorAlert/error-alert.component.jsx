import React from 'react';

const ErrorAlert = () => (
  <>
    <h1 className="t1">Oooopsss...</h1>
    <h3>An error occurred. Please try again later!</h3>
  </>
);

export default ErrorAlert;
