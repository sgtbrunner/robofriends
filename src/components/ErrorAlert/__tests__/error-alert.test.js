import React from 'react';
import { shallow } from 'enzyme';
import ErrorAlert from '../error-alert.component';

const mountComponent = () => shallow(<ErrorAlert />);

describe('ErrorAlert component', () => {
  describe('render', () => {
    it('expect ErrorAlert component to match snapshot', () => {
      const wrapper = mountComponent();
      expect(wrapper).toMatchSnapshot();
    });
  });
});
