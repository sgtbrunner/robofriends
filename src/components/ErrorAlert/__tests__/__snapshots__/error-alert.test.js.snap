// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorAlert component render expect ErrorAlert component to match snapshot 1`] = `ShallowWrapper {}`;
